rpc_jobs/rpc_upgrades.yml

```diff
@@ -270,8 +270,38 @@
     scenario:
       - swift
     action:
+      - mitaka_to_r14.current_snapcap
+      - r13.0.0_to_r14.current_snapcap
+      - r13.0.2_to_r14.current_snapcap
+      - r13.1.2_to_r14.current_snapcap
       - r13.1.3_to_r14.current_snapcap
+      - r13.1.4_to.r14.current_snapcap
+      - liberty_to_r14.current_snapcap
+      - r12.2.8_to_r14.current_snapcap
+      - r12.2.5_to_r14.current_snapcap
+      - r12.2.2_to_r14.current_snapcap
+      - r12.1.2_to_r14.current_snapcap
+      - kilo_to_r14.current_snapcap
     jira_project_key: "RLM"
+    jobs:
+      - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+    CRON: "{CRON_MONTHLY}"
```

Comment:
If these are for one-off image generation, then CRON can be set to an empty string to make these jobs all manually executed. That way you can build them on demand, but leave them in place.